en/components/calendar.md

```diff
@@ -263,6 +358,8 @@ When a day inside the current month is focused:
 | `formatOptions` | `Object` | The format options passed along with the `locale` property used for formatting the dates. Defaults are { day: 'numeric', month: 'short', weekday: 'short', year: 'numeric' }. |
 |`formatViews`| `Object`| Controls whether the date components in the different calendar views should be formatted according to the provided locale and formatOptions. Defaults are { day: false, month: true, year: false }. Does not affect rendering in the header. |
 | `vertical` | `boolean` | Controls the layout of the calendar component. When vertical is set to true the calendar header will be rendered to the side of the calendar body.|
+| `disabledDates` | `array` | Gets\Sets the disabled dates descriptors. |
```

Comment:
Should be `Gets/Sets` instead of `Gets\Sets`.

Reply:
Done